src/jdk.jlink/share/classes/jdk/tools/jmod/resources/jmod.properties

```diff
@@ -82,6 +82,9 @@ main.opt.date=Date and time for the timestamps of entries, specified in ISO-8601
 
 main.opt.cmdfile=Read options from the specified file
 
+main.opt.compression-level=Compression level to use when compressing the JMOD archive.\
```

Comment:
Hello Aleksey, should this instead say "Compression level to use when creating the JMOD archive."?

Reply:
Right. Done in new commit (to be published soon).